src/components/ProfileMini.js

```diff
@@ -133,13 +134,22 @@ const ProfileMini = ({
 
   return (
     <Fragment>
-      <DialogAddMember
-        address={address}
-        handleAddMember={handleAddMember}
-        handleClose={handleClose}
-        isOpen={isOpen}
-        username={username}
-      />
+      {isSharedWalletCreation ? (
```

Comment:
I think the name of this variable is misleading because it should also chose dialogAddmember if you are in add member view. Thus not only when you create sharedwallet. 
Better name is isModeAddNotTrust or something like that

Reply:
Maybe `isAddMemberMode`?
We can talk about making code more dry. I don't think we copy same logic several times as we have this `Finder` component which is used in two places. That's why wrapping `Finder` and `DialogAddMember` component as a new component doesn't bring too much value for me. Still there will have to be a logic which replace `DialogAddMember` component as it is use in one place not both. But I may miss something.. 
So discussion is welcomed 🌞